modules/update-service-install-web-console.adoc

```diff
@@ -0,0 +1,35 @@
+[id="update-service-install-web-console_{context}"]
+= Installing the OpenShift Update Service Operator by using the web console
+
+You can use the {product-title} web console to install the OpenShift Update Service Operator.
+
+.Procedure
+
+To install the OpenShift Update Service Operator using the {product-title} web console:
+
+. In the {product-title} web console, click *Operators* -> *OperatorHub*.
+
+. Choose *OpenShift Update Service* from the list of available Operators, and click *Install*.
+
+[NOTE]
+====
+Enter `Update Service` into the *Filter by keyword...* field to find the operator faster.
+====
+
+. `v1` will be selected as the *Update Channel* since it is the only channel available in this release.
+
+. Select *A specific namespace on the cluster* under *Installation Mode*.
+
+. Select a namespace for *Installed Namespace* or accept the recommended namespace `openshift-update-service`.
+
+. Select an *Approval Strategy*:
++
+** The *Automatic* strategy allows Operator Lifecycle Manager (OLM) to automatically update the Operator when a new version is available.
++
+** The *Manual* strategy requires a cluster administrator to approve the Operator update.
```

Comment:
A `Manual approval required` alert pops up when Manual strategy is selected. Need to click Approve to proceed the operator installation.
![Screenshot from 2021-03-25 12-24-32](https://user-images.githubusercontent.com/22511421/112419040-9cfd6e80-8d65-11eb-9364-db826af70062.png)

Reply:
They'll know this from after reading the prereq ` Installing Operators in your namespace` so see no reason to repeat here.